src/EditModel/event-program/tracker-program/assign-tracked-entity-attributes/AssignAttributes.js

```diff
@@ -152,17 +98,26 @@ function AssignAttributes(props, { d2 }) {
 
     // Assign existing attributes
     assignedItemStore.setState(
-        props.items.map(a => a.trackedEntityAttribute.id),
+        props.assignedAttributes.map(a => a.trackedEntityAttribute.id),
     );
 
+    const onMoveAttributes = (newAttributesOrderIds) => {
+        assignedItemStore.setState(newAttributesOrderIds);
+        // need to update this.props.assignedAttributes to reflect new order in epics
```

Comment:
Trivial, but consider adding `TODO` to such comments. IDEs like IntelliJ automatically keep track of TODOs in comments and for the rest they are easily searchable with tools like `ack` and `grep`.